src/kbmod/reprojection.py

```diff
@@ -31,7 +34,59 @@ def reproject_raw_image(image, original_wcs, common_wcs, obs_time):
         for footprint[i][j], it's 1 if there is a corresponding reprojected
         pixel and 0 if there is no data.
     """
-    image_data = CCDData(image.image, unit="adu")
+    return _reproject_image(image.image, original_wcs, common_wcs)
+
+
+def reproject_ndarray_image(image, original_wcs, common_wcs):
```

Comment:
Optional thought: These two functions could be collapsed into one by doing something like:
```
if type(image) is RawImage:
    image = image.image
```